force-app/main/default/classes/Homework.cls

```diff
@@ -63,38 +82,51 @@ public with sharing class Homework {
     {
         // Declare two Integers, with the values 6 and 7, in a single 
         // statement.
+        Integer firstValue = 6, secondValue = 7;
     }
 
     public void booleanFromString()
     {
         // Create a boolean whose value is initialised from the text
         // ‘true’.
+        
+        Boolean myBooleanValue = true;
+
     }
 
     public void reverse()
     {
         // Create a string with the value ‘1234567890’ 
-        
+        String leftToRight = '1234567890';
         // Use a built-in method to convert to ‘0987654321’
+        String rightToLeft = leftToRight.reverse();
     }
 
     public void findApex(String candidate) 
     {
         // Find the position of the substring “Apex” in candidate
+        String newCandidate = 'Random Apex Guy';
```

Comment:
The comment is asking you to find the location in the parameter `candidate` - why are you creating a new String and using that?